ashley/src/com/badlogic/ashley/core/Entity.java

```diff
@@ -75,6 +75,18 @@ public Entity add (Component component) {
 		}
 		return this;
 	}
+	
+	/**
+	 * Adds all given {@link Component}s to this Entity. See {@link Entity#add(Component)}.
```

Comment:
Could you add that this will allocate memory and generate work for the GC? 

http://stackoverflow.com/questions/7452513/java-varargs-allocates-array-objects